src/main.rs

```diff
@@ -881,10 +954,12 @@ fn get_setup() -> Setup {
                     exit(1);
                 }
             })
-            .unwrap_or_else(|| match mixer_type.as_deref() {
-                #[cfg(feature = "alsa-backend")]
-                Some(AlsaMixer::NAME) => 0.0, // let alsa query the control
-                _ => VolumeCtrl::DEFAULT_DB_RANGE,
+            .unwrap_or_else(|| {
+                if is_alsa_mixer {
+                    0.0
+                } else {
+                    VolumeCtrl::DEFAULT_DB_RANGE
```

Comment:
Though not entirely in the scope of this PR, should I have added an `impl Default for VolumeCtrl` before? So we could do `VolumeCtrl::default().db_range` or something like that here.

Reply:
All of the config stuff could use a little love but that's a diffrent PR for after you finish the new api stuff.

Reply:
That's fine.